src/cmd/common/viper.go

```diff
@@ -168,25 +169,24 @@ func isVersionCmd() bool {
 }
 
 // PrintViperConfigUsed informs users when Zarf has detected a config file.
-func PrintViperConfigUsed(ctx context.Context) {
+func PrintViperConfigUsed(ctx context.Context) error {
 	l := logger.From(ctx)
 
 	// Only print config info if viper is initialized.
 	vInitialized := v != nil
 	if !vInitialized {
-		return
+		return nil
 	}
 	var notFoundErr viper.ConfigFileNotFoundError
 	if errors.As(vConfigError, &notFoundErr) {
-		return
+		return nil
 	}
 	if vConfigError != nil {
-		l.Error("unable to load config file", "error", vConfigError)
-		return
-	}
-	if cfgFile := v.ConfigFileUsed(); cfgFile != "" {
-		l.Info("using config file", "location", cfgFile)
+		return fmt.Errorf("unable to load config file: %w", vConfigError)
 	}
+	cfgFile := v.ConfigFileUsed()
```

Comment:
We already return if a config file is found, so no need to check for an empty string